lib/IRGen/IRGenModule.h

```diff
@@ -998,9 +998,9 @@ class IRGenModule {
   
 //--- Globals ---------------------------------------------------------------
 public:
-  std::pair<llvm::GlobalVariable *, llvm::Constant *>
-  createStringConstant(StringRef Str, bool willBeRelativelyAddressed = false,
-                       StringRef sectionName = "");
+  std::pair<llvm::GlobalVariable *, llvm::Constant *> createStringConstant(
+      StringRef Str, bool willBeRelativelyAddressed = false,
+      StringRef sectionName = "", const llvm::Twine &name = "");
```

Comment:
This is _really_ not a good idea.  `Twine` is very easy to mishandle, please use a `StringRef`.

Reply:
Huh, no idea why I did that. Will fix, of course!